packages/grafana-data/src/field/fieldState.ts

```diff
@@ -30,39 +30,39 @@ export function getFrameDisplayTitle(frame: DataFrame, index?: number) {
   return `Series (${index})`;
 }
 
-export function getFieldTitle(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
-  const existingTitle = field.state?.title;
+export function getFieldDisplayName(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
+  const existingTitle = field.state?.displayName;
 
   if (existingTitle) {
     return existingTitle;
   }
 
-  const title = calculateFieldTitle(field, frame, allFrames);
+  const displayName = calculateFieldDisplayName(field, frame, allFrames);
   field.state = {
     ...field.state,
-    title,
+    displayName,
   };
 
-  return title;
+  return displayName;
 }
 
 /**
- * Get an appropriate display title. If the 'title' is set, use that
+ * Get an appropriate display name. If the 'title' is set, use that
  */
-function calculateFieldTitle(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
+function calculateFieldDisplayName(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
   const hasConfigTitle = field.config?.title && field.config?.title.length;
 
-  let title = hasConfigTitle ? field.config!.title! : field.name;
+  let displayName = hasConfigTitle ? field.config!.title! : field.name;
 
   if (hasConfigTitle) {
-    return title;
+    return displayName;
   }
 
   // This is an ugly exception for time field
   // For time series we should normally treat time field with same name
   // But in case it has a join source we should handle it as normal field
   if (field.type === FieldType.time && !field.labels) {
-    return title ?? 'Time';
+    return displayName ?? 'Time';
```

Comment:
nit: Do we have a const for the magic string `'Time'`?

Reply:
yeah, would be great

Reply:
added new constant :)